oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -231,7 +231,7 @@ export default class Map extends React.PureComponent {
         });
       }
     };
-    this.map.addControl(this.scaleViewer);
```

Comment:
fails because map is undefined because we haven't created it yet lol. Line ~679 is the correct placement https://github.com/DFO-Ocean-Navigator/Ocean-Data-Map-Project/blob/9c582c0d90746ed28755b944ec59b9d061fd2c4c/oceannavigator/frontend/src/components/Map.jsx#L679